easybuild/easyconfigs/a/AUGUSTUS/AUGUSTUS-3.3.2-intel-2017b-Python-2.7.14.eb

```diff
@@ -0,0 +1,58 @@
+easyblock = 'ConfigureMake'
+
+name = 'AUGUSTUS'
+version = '3.3.2'
+versionsuffix = '-Python-2.7.14'
+
+homepage = 'http://bioinf.uni-greifswald.de/augustus/'
+description = "AUGUSTUS is a program that predicts genes in eukaryotic genomic sequences"
+
+toolchain = {'name': 'intel', 'version': '2017b'}
+
+source_urls = [
+    'http://bioinf.uni-greifswald.de/augustus/binaries/',
+    'http://bioinf.uni-greifswald.de/augustus/binaries/old/',
+]
+sources = [SOURCELOWER_TAR_GZ]
+patches = [
+    '%(name)s-%(version)s_fix-hardcoding.patch',
+    '%(name)s-%(version)s_additional_fix_hardcoding.patch',
+]
+checksums = [
+    '989a95fe3a83d62af4d323a9727d11b2c566adcf4d789d5d86d7b842d83e7671',  # augustus-3.3.2.tar.gz
+    '96e8d72b67e820dca0d6fdbda5b9735dc8259e27fe06167462f2bdbfef5ed7d7',  # AUGUSTUS-3.3.2_fix-hardcoding.patch
+    # AUGUSTUS-3.3.2_additional_fix_hardcoding.patch
+    'c42b14d52f4b98584ac9bcf2568c73feca987f563d40628be143ada326e9dccc',
+]
+
+dependencies = [
+    ('zlib', '1.2.11'),
+    ('Boost', '1.65.1', versionsuffix),
```

Comment:
If Boost with Python is really needed, then there should probably be an explicit Python dependency.
And you should use Boost.Python instead of Boost

Reply:
same comments for the other AUGUSTUS config of course.

Reply:
@akesandgren I agree with you on this. However since `Boost.Python` was available after we had already installed `Boost` with Python support on our cluster, we still have the old setup for the 2017b toolchain. This also explains why there is the Python versionsuffix you asked about.

Reply:
ok, as long as there is a good reason :-)